src/main/java/org/quartzpowered/network/pipeline/MinecraftChannelInitializer.java

```diff
@@ -35,9 +35,12 @@
 
 @Singleton
 public class MinecraftChannelInitializer extends ChannelInitializer<SocketChannel> {
-    @Inject private CodecFactory codecFactory;
-    @Inject private PacketHandler handler;
-    @Inject private NoopHandler noop;
+    @Inject
+    private CodecFactory codecFactory;
+    @Inject
+    private PacketHandler handler;
+    @Inject
+    private NoopHandler noop;
 
```

Comment:
These are supposed to be on one line, please allow one line annotations in your formatting options and update the PR.